cts-java/src/test/java/org/ga4gh/cts/api/Utils.java

```diff
@@ -193,19 +192,29 @@ public static String getReadGroupIdForName(Client client, String name) throws Av
      */
     public static List<ReadGroupSet> getAllReadGroupSets(Client client)
             throws AvroRemoteException {
-        final SearchReadGroupSetsRequest readGroupSetsReq =
-                SearchReadGroupSetsRequest
-                        .newBuilder()
-                        .setDatasetId(TestData.getDatasetId())
-                        .build();
-        final SearchReadGroupSetsResponse readGroupSetsResp =
-                client.reads.searchReadGroupSets(readGroupSetsReq);
-        assertThat(readGroupSetsResp).isNotNull();
-        final List<ReadGroupSet> readGroupSets = readGroupSetsResp.getReadGroupSets();
-        assertThat(readGroupSets).isNotEmpty().isNotNull();
-        return readGroupSets;
-    }
 
+        final List<ReadGroupSet> result = new LinkedList<>();
```

Comment:
This can probably be refactored to be a call to `getAllReadGroupSetsForName(client, null)`.